### PR TITLE
use pyright instead of mypy

### PR DESCRIPTION
**Pull request by lucabello**
`src/kubernetes_service.py` has had its import modified as the `kubernetes` pypi package does not have type hints, and from their open issues it doesn't seem like they're coming anytime soon :)

### CODE REVIEW

File: src/kubernetes_service.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
 from typing import List, Tuple
 
-import kubernetes
```
Comment (PietroPasotti) [inline src/kubernetes_service.py:9]: [clarification] why is this change necessary (or why is it part of this pr?)

**Review by PietroPasotti — APPROVED** [review]
looks good!

File: src/kubernetes_service.py (hunk 1 above)
Comment (lucabello) [reply src/kubernetes_service.py:9]: It's necessary because the **kubernetes** pypi package doesn't use typing much, and the linter struggles to find class properties if you don't specify it like that !
